platforms/core-configuration/configuration-problems-base/src/main/kotlin/org/gradle/internal/cc/impl/problems/JsonModelWriter.kt

```diff
@@ -18,20 +18,17 @@ package org.gradle.internal.cc.impl.problems
 
 class JsonModelWriter(val modelWriter: JsonWriter) {
     fun beginModel() {
-        with(modelWriter) {
+        modelWriter.run {
```

Comment:
❓ This is probably not necessary; why did you change that?

Reply:
As far as I can tell, `run` is used more consistently in our code base than `with`.